### PR TITLE
Added `php console make:request` command

### PR DESCRIPTION
**Pull request by rumur**
Hi @jlambe !

Today played around with 2.0, with all new features, a great job is done!

Also wanted to try [Laravel Request Forms](https://laravel.com/docs/5.8/validation#form-request-validation) in action, noticed that you have `FormRequest` already in the Core,  but 'make:request' command was missed, so I've added, hope you don't mind :)

### CODE REVIEW

**Comment by jlambe** [comment]
Hi @rumur, nice addition but we still need to register the command on the core console kernel. Could you take a look to add it as a development command within the core `ConsoleServiceProvider`: https://github.com/themosis/framework/blob/master/src/Core/Providers/ConsoleServiceProvider.php#L80

**Comment by rumur** [comment]
@jlambe Yes, sure.

File: src/Core/Providers/ConsoleServiceProvider.php [hunk 1]
```diff
@@ -96,7 +97,8 @@ class ConsoleServiceProvider extends ServiceProvider
         'ThemeInstall' => 'command.theme.install',
         'VendorPublish' => 'command.vendor.publish',
         'Serve' => 'command.serve',
-        'WidgetMake' => 'command.widget.make'
+        'WidgetMake' => 'command.widget.make',
+        'RequestMake' => 'command.request.make',
```
Comment (jlambe) [inline src/Core/Providers/ConsoleServiceProvider.php]: The code is ok for me. Just that the command lists are organized in alphabetical order in this file. Could you move the `RequestMake` command in the list?

File: src/Core/Providers/ConsoleServiceProvider.php [hunk 2]
```diff
@@ -542,6 +544,18 @@ protected function registerWidgetMakeCommand($abstract)
         });
     }
 
+    /**
+     * Register the make:request command.
+     *
+     * @param string $abstract
+     */
+    protected function registerRequestMakeCommand($abstract)
```
Comment (jlambe) [inline src/Core/Providers/ConsoleServiceProvider.php]: The same goes for the declared method, the `registerRequestMakeCommand` should not be the last method call. Methods are also organized in alphabetical order in the file. Thank you.

File: src/Core/Providers/ConsoleServiceProvider.php (hunk 1 above)
Comment (rumur) [reply src/Core/Providers/ConsoleServiceProvider.php]: of course, no problem

File: src/Core/Providers/ConsoleServiceProvider.php (hunk 2 above)
Comment (rumur) [reply src/Core/Providers/ConsoleServiceProvider.php]: yes, sure.

**Pull request merged**